docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
       - postgres
   mysql:
     image: mysql
+    platform: linux/amd64
```

Comment:
this was needed for Mac M1 machines, and I figured it's useful unless mysql pushes out an arm image.